crates/loader/src/local/tests.rs

```diff
@@ -252,7 +252,7 @@ async fn test_invalid_url_in_allowed_http_hosts_is_rejected() -> Result<()> {
 
     let temp_dir = tempfile::tempdir()?;
     let dir = temp_dir.path();
-    let app = from_file(MANIFEST, dir, &None).await;
+    let app = from_file(MANIFEST, dir, &None, vec![]).await;
```

Comment:
Should we have a test for when the include list is non-empty?  (And for error cases.)